cmd/namespace.go

```diff
@@ -56,13 +102,27 @@ Please note that at this time all namespaces created in the registry are world-r
 func createNamespace(opts namespaceOptions) error {
 	namespaceName := opts.args[0]
 
-	_, err := api.CreateNamespace(opts.cl, namespaceName, opts.args[2], strings.ToUpper(opts.args[1]))
+	if !opts.noPrompt {
+		fmt.Printf(`You are creating a namespace called "%s".
+
+This is the only namespace permitted for your %s organization, %s.
+
+To change the namespace, you will have to contact CircleCI customer support.
+
+`, namespaceName, strings.ToLower(opts.args[1]), opts.args[2])
+	}
+
+	confirm := fmt.Sprintf("Please confirm the namespace you wish to create: `%s`.", namespaceName)
```

Comment:
<img width="633" alt="screen shot 2019-01-08 at 14 19 36" src="https://user-images.githubusercontent.com/555144/50811137-7dc6c200-1350-11e9-8683-989069ebc4eb.png">

A very similar issue to https://github.com/CircleCI-Public/circleci-cli/pull/260#discussion_r245878335 - I'd at least remove that `.`.